codegen/java-gen/src/main/scala/com/lightbend/akkasls/codegen/java/EntityServiceSourceGenerator.scala

```diff
@@ -211,7 +211,9 @@ object EntityServiceSourceGenerator {
               Nil,
               emptyDoc
             )(
-              "return" <+> qualifiedType(state.fqn) <> ".getDefaultInstance();"
+              "throw new UnsupportedOperationException" <> parens(
+                dquotes("Not implemented yet, replace with your empty entity state")
+              ) <> semi
```

Comment:
These are codegen changes requested in #132: 
 * dont' generate a working default empty state
 * name the command and events after their type instead of `command` and `event`